osu.Game/Beatmaps/DummyWorkingBeatmap.cs

```diff
@@ -26,7 +26,7 @@ public DummyWorkingBeatmap(OsuGameBase game)
                     Title = "no beatmaps available!",
                     Author = "no one",
                 },
-                BeatmapSet = new BeatmapSetInfo(),
+                BeatmapSet = new BeatmapSetInfo { DeletePending = true }, //let song select going to status that no beatmaps available

```

Comment:
How about "let song select show that no beatmaps are available if this is the only one"?

Reply:
Actually this doesn't have any effect in general as far as I can tell?

Reply:
your suggestion sounds good